test/test_bundle_importer.js

```diff
@@ -67,6 +66,15 @@ describe('Bundle Importer', function() {
   });
 
   describe('Public methods', function() {
+    describe('importBundleYAML', function() {
+      it('calls fetchDryRun with yaml', function() {
```

Comment:
This test actually doesn't do a whole lot. It's just checking that calling the public `importBundleYAML` method calls the `fetchDryRun` method. The `fetchDryRun` method tests are further down in the suite.